src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -712,4 +712,7 @@ Examples:
   <data name="TemplatePackageCoordinator_Update_Info_UpdateSingleCommandHeader" xml:space="preserve">
     <value>To update the package use:</value>
   </data>
+  <data name="OptionDescriptionNoUpdateCheck" xml:space="preserve">
+    <value>Disables check for template package update performed after template instantiation.</value>
```

Comment:
Maybe:
```suggestion
    <value>Disables checking for template package updates after instantiating a template.</value>
```

Do we check for updates "after" template instantiation? I thought we did it "before".

Reply:
To be precise we do it in parallel now, probably "on" will be better preposition.